book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
+end
```

Comment:
Kindly make sure that rentals has setters and getters method, and inside the add rental method the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association.
Example: 
```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```